lib/actions/updateDispatchDate.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+const fetch = require('node-fetch');
+const requestOptions = require('../requestOptions.js');
+
+exports.process = function(msg, cfg) {
+  let self = this;
+  let orderId = msg.body.webhook.data.id;
+  let dispatchDate = msg.body.electio.dispatchDate;
+
+  function updateOrder(authenticationHeaders) {
+     let requestBody = {
```

Comment:
Move this variable definition inside the `if` statement. It is not needed if `dispatchDate` is `null`.